trail/query/models.py

```diff
@@ -1,3 +1,5 @@
-# from django.db import models
+from django.db import models
 
-# Create your models here.
+
+class Observation(models.Model):
```

Comment:
Please add a docstring to this class. You can follow the guide at https://developer.lsst.io/python/numpydoc.html#documenting-classes